### ISSUE TITLE
Miss app icon in login page for iPhone (but present in iPad)

### ISSUE THREAD

**Issue by magicboker** [dropbox/dropbox-sdk-obj-c#63]
My app used to show app icon in the login page by using v1 API, but it missed app icon when using v2 API with this SDK for iPhone.
Like this
![missing app icon](https://dl.dropboxusercontent.com/u/21305787/dropbox_login_missing_app_icon.png)

**Comment by greg-db** [comment]
Thanks for the report! The mobile version of the page only shows the Dropbox logo during sign in, but the app's icon is shown next during the app authorization. We'll consider this a request for showing the app icon during sign in on mobile as well.

**Comment by magicboker** [comment]
Why the app icon was shown in the WebAuth login page on iPad with exactly the same code?
Did your web server detect the client type and shows different result?

**Comment by greg-db** [comment]
Yes, the iPad has more room so it gets the desktop version of that page.

**Comment by magicboker** [comment]
You don't need to have extra place to show my app icon.
Where the dropbox icon is shown in SDK v2 used to show my app icon in SDK v1.

**Comment by greg-db** [comment]
Thanks! I've sent this along to the team to see if we can make that change for v2.

**Comment by magicboker** [comment]
Thanks, look forward to the change!
